core/templates/dev/head/pages/collection_player/CollectionLocalNav.js

```diff
@@ -0,0 +1,21 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Controller for the local navigation in the collection view.
+ */
+
+oppia.controller('CollectionLocalNav', ['$scope', function($scope) {
+  $scope.canEdit = GLOBALS.canEdit;
+}]);
```

Comment:
This might seem as to small controller, but it will probably get bigger later when we remove GLOBALS.